ephios/extra/templates/extra/widgets/recurrence_picker.html

```diff
@@ -0,0 +1,219 @@
+                                   v-model="rule.month_mode" value="bysetpos" required>
+                        </div>
+                        <div class="col-3">
+                            <select class="form-select" v-model="rule.bysetpos"
+                                    @focus="rule.month_mode='bysetpos'">
+                                <option value="1">{% translate "First" %}</option>
+                                <option value="2">{% translate "Second" %}</option>
+                                <option value="3">{% translate "Third" %}</option>
+                                <option value="4">{% translate "Fourth" %}</option>
+                                <option value="-1">{% translate "Last" %}</option>
+                            </select>
+                        </div>
+                        <div class="col-3">
+                            <select class="form-select" v-model="rule.byweekday"
+                                    @focus="rule.month_mode='bysetpos'">
+                                <option v-for="(label, pos_index) in weekdays" :value="pos_index">[[ label ]]</option>
+                            </select>
+                        </div>
+                        <div class="col-1">
+                            <span>{% translate "in" context "month" %}</span>
+                        </div>
+                        <div class="col-3">
+                            <select class="form-select" v-model="rule.bymonth"
+                                    @focus="rule.month_mode='bysetpos'">
+                                <option v-for="month in months" :value="month.id">[[ month.long ]]</option>
+                            </select>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="card p-2">
+                    <div class="form-check mb-2">
+                        <input :id="'until' + index" class="form-check-input mt-2" type="radio" name="end_mode" v-model="rule.end_mode"
+                               value="UNTIL" required>
+                        <label class="form-check-label" :for="'until' + index">
+                            {% translate "until" %}
+                        </label>
+                        <input type="date" class="form-control w-25 d-inline-block ms-2 me-2 form-control-sm"
+                               v-model="rule.until" @focus="rule.end_mode='UNTIL'"
+                               :required="rule.end_mode=='UNTIL'">
+                        <input v-if="pickHour" type="time" class="form-control form-control-sm d-inline-block w-25"
+                               v-model="rule.UNTIL_TIME" @focus="rule.end_mode='UNTIL'">
+
+                    </div>
+
+                    <div class="form-check">
+                        <input :id="'until' + index" class="form-check-input mt-2" type="radio" name="end_mode" v-model="rule.end_mode"
```

Comment:
```suggestion
                        <input :id="'count' + index" class="form-check-input mt-2" type="radio" name="end_mode" v-model="rule.end_mode"
```

Otherwise, the labels "for" and "occurrences" do not work